README.md

```diff
@@ -28,14 +29,81 @@ uv run orgwarden list-repos <org_url>
 ```
 
 ### Audit
-Runs [RepoAuditor](https://github.com/gt-sse-center/RepoAuditor) tooling. If the provided url points to a GitHub repository, RepoAuditor will run against said repository. If the provided url points to a GitHub organization, RepoAuditor will run against all public, non-forked repositories within said organization. The `gh-pat` option is a GitHub Personal Access Token (PAT) that is used by RepoAuditor for increased functionality.
+Runs [RepoAuditor](https://github.com/gt-sse-center/RepoAuditor) tooling. If the provided url points to a GitHub repository, RepoAuditor will run against said repository. If the provided url points to a GitHub organization, RepoAuditor will run against all public, non-forked repositories within said organization. The `gh-pat` option is a GitHub Personal Access Token (PAT) that RepoAuditor requires for full funcionality.
+
+**Note** - If a PAT is not provided, `audit` will run with limited functionality **and will always exit with a non-zero exit code.**
 
-*RepoAuditor currently only supports classic PATs.*
+*Due to an issue with the GitHub API, OrgWarden currently only supports Classic PATs.*
```

Comment:
Can you include a link to the GitHub issue that you created? With that link, curious people can see if the issue has been resolved.